src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -145,4 +105,44 @@ public int size() {
     public boolean isEmpty() {
         return size == 0;
     }
+
+    private T unlink(Node<T> deletedNode) { // Для зміни зв'язків
+        Node<T> nextNode = deletedNode.next;
+        Node<T> prevNode = deletedNode.prev;
+        if (prevNode == null) {
+            first = nextNode;
+        } else {
+            prevNode.next = nextNode;
+        }
+
+        if (nextNode == null) {
+            last = prevNode;
+        } else {
+            nextNode.prev = prevNode;
+        }
+        size--;
+        return deletedNode.item;
+    }
+
+    private void checkPosition(int index) {
+        if (index < 0 || index >= size) {
```

Comment:
imagine we have 10 elements in list. indexes are 0-9. Can I insert new node on index 10? This check says I cannot. Think about it and try to add this method again where it didn't work

Reply:
Checking allows you to put an item on the 10th index.

But you made me find my mistake. 
When I passed (index -1), it wasn't < 0.

I created new check method for add().